### ISSUE TITLE
ScalaLintBear: Add MavenRequirement

### ISSUE THREAD

**Issue by jayvdb** [coala/coala-bears#1654]

**Comment by Shade5** [comment]
I'm on it

**Comment by yash-nisar** [comment]
@Shade5 The release is within a few hours so can you initiate a PR asap ?

**Comment by Shade5** [comment]
@yash-nisar I tried working on it but many tests are failing. I'm also flooded with projects and finals :sweat_smile:. I don't think I'll be able to push a working PR before the release. I'm sorry.

**Comment by jayvdb** [comment]
@Shade5 , I hope you dont mind. You have a few other PRs to close up.

**Comment by Shade5** [comment]
@jayvdb Not at all, please assign it to someone who's free. I'm quite occupied for a few more days with my finals. If no one resolves it by then, I'll assign it to myself and continue :+1:
